src/components/Announcements.js

```diff
@@ -21,14 +46,87 @@ class Announcements extends React.Component {
     this.setState({ postValues: json });
   }
 
+  async buttonClick() {
+    const response = await fetch("http://localhost:3001/announcements", {
+      method: "POST",
+      headers: {
+        Accept: "application/json",
+        "Content-type": "application/json",
+        authorization: localStorage.get("token")
+      },
+      body: JSON.stringify({
+        title: this.state.title,
+        info: this.state.info
+      })
+    });
+    const json = await response.json();
+    this.setState({
+      creating: false,
+      postValues: [{ id: json.id, user_id: null, title: this.state.title, info: this.state.info, time: null }].concat(this.state.postValues)
+    })
+  }
+
   render() {
     return (
-      <div className="header">
+      <div className="header" >
         <h1 className="Announcements-section">ANNOUNCEMENTS </h1>
-        {this.state.postValues.map(item => (
-          <Post key={item.id} {...item} />
-        ))}
-      </div>
+        <div className="dropdown-button">
+          <Button
+            primary
+            onClick={() =>
+              this.setState({
+                creating: true
+              })
+            }
+          >
+            Make new announcement
+          </Button>
+        </div>
+        <div>
+          {this.state.creating && (
+            <div className="input">
+              <form>
+                <textarea
+                  className="title"
+                  id="story"
+                  name="story"
+                  resize="vertical"
+                  value={this.state.input}
```

Comment:
I don't think this line (or value in the other textarea) is doing anything because this state variable doesn't exist. It can be removed